python-template/{{cookiecutter.placeholder_repo_name}}/setup.py

```diff
@@ -36,17 +36,45 @@ def load_requirements(*requirements_paths):
     with -c in the requirements files.
     Returns a list of requirement strings.
     """
+    # e.g. {"django": "Django", "openedx-events": "openedx_events"}
```

Comment:
Just to be super de dooper clear
```suggestion
    # e.g. {"django": "Django", "openedx-events": "openedx_events", "confluent-kafka":"confluent-kafka[avro]"}
```